remote-trait-object/src/packet.rs

```diff
@@ -80,8 +81,7 @@ const SLOT_CALL_OR_RETURN_INDICATOR: SlotId = SlotId(1000);
 #[repr(C)]
 struct PacketHeader {
     pub slot: SlotId,
-    // FIXME: use integer indices for the service
-    pub target_service_name: [u8; 100],
+    pub object: ServiceObjectId,
```

Comment:
How about `service_object` or `service_object_id`?
`Object` is a too common name.